annotations/update_pending_annotations.py

```diff
@@ -0,0 +1,152 @@
+import synapseclient
+from synapseclient import Table, RowSet
+import argparse
+import pandas as pd
+from attribute_dictionary import PUBLICATION_DICT
+
+
+### Login to Synapse ###
```

Comment:
This comment should ideally go into the function as a docstring instead, e.g.

```suggestion
def login():
   """Login to Synapse."""
```

That way, if/when we were to auto-generate docs from the script, the tool will know where the function description is.

Reply:
Thanks! I edited this!